lte/gateway/c/session_manager/SessionStateEnforcer.h

```diff
@@ -159,6 +159,23 @@ class SessionStateEnforcer {
       std::unique_ptr<SessionState>& session, SessionFsmState target_state,
       SessionStateUpdateCriteria* session_uc);
 
+  /*
+   * Acquire and update session rules based on the IMSI
+   *

```

Comment:
Can we also add a doxygen style doc here?
```
/**
* @brief TODO
* @param gnb_teid_get: TODO
* @param upf_teid_get: TODO
* @returns TODO
*/
```

Reply:
Added doxygen style doc here